### PR TITLE
Add JerryScript logo.

### PR DESCRIPTION
**Pull request by lemmaa**
JerryScript-DCO-1.0-Signed-off-by: Sung-Jae Lee sjlee@mail.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+![](https://github.com/lemmaa/jerryscript/blob/master/LOGO.png)
 # JerryScript: JavaScript engine for Internet of Things
```
Comment (akiss77) [inline README.md]: Shouldn't this url point into the Samsung/jerryscript main repo? (Once this PR gets merged, it will become valid.)

Comment (lemmaa) [inline README.md]: @akiss77 Thanks, I have updated the PR!

**Comment by akiss77** [comment]
LGTM (informally)

**Comment by zherczeg** [comment]
LGTM

**Comment by lemmaa** [comment]
@egavrin @sand1k @ruben-ayrapetyan How about you?

**Comment by sand1k** [comment]
LGTM.

**Comment by LaszloLango** [comment]
Landed (41f6cbb3aca8260c6cc7eef064ec45946041252f)

**Pull request closed without merging**